services/rngService.js

```diff
@@ -4,24 +4,48 @@ const getRandomValue = (mean, stdev, method) => {
   return method(1, mean, stdev);
 };
 
-const normalDistribution = (mean, stdev, modifier = 1) => {
-  return Math.floor(
-    parseInt(
-      getRandomValue(mean, stdev, numbers.random.distribution.normal) *
+const checkIfNegative = (value) => {
+  return value <= 0 ? 0 : value;
+};
+
+const normalDistribution = (mean, stdev, modifier = 1, checkNegative = false) => {
+  if (checkNegative) {
+    return checkIfNegative(Math.floor(
+      parseInt(
+        getRandomValue(mean, stdev, numbers.random.distribution.normal) *
         modifier,
-      10
-    )
-  );
+        10
+      )
+    ));
+  } else {
+    return Math.floor(
+      parseInt(
+        getRandomValue(mean, stdev, numbers.random.distribution.normal) *
+        modifier,
+        10
+      )
+    );
+  }
 };
 
-const logNormalDistribution = (mean, stdev, modifier = 1) => {
-  return Math.floor(
-    parseInt(
-      getRandomValue(mean, stdev, numbers.random.distribution.logNormal) *
+const logNormalDistribution = (mean, stdev, modifier = 1, checkNegative = false) => {
+  if (checkNegative) {
+    return checkIfNegative(Math.floor(
+      parseInt(
+        getRandomValue(mean, stdev, numbers.random.distribution.logNormal) *
+        modifier,
+        10
+      )
+    ));
+  } else {
+    return Math.floor(
+      parseInt(
+        getRandomValue(mean, stdev, numbers.random.distribution.logNormal) *
         modifier,
```

Comment:
Same here, may as well get the number first and apply `checkIfNegative()` if the param is set.